uclapi.openapi.json

```diff
@@ -0,0 +1,4256 @@
+        "properties" : {
+          "description_2" : {
+            "type" : "string",
+            "example" : "",
+            "description": "Description 2."
+          },
+          "floor" : {
+            "type" : "string",
+            "example" : "False",
+            "description": "Whether the device is on the floor"
+          },
+          "y_pos" : {
+            "type" : "string",
+            "example" : "14893.0",
+            "description": "The Y coordinate position of the sensor on the map."
+          },
+          "description_3" : {
+            "type" : "string",
+            "example" : "",
+            "description": "Description 3."
+          },
+          "device_type" : {
+            "type" : "string",
+            "example" : "Desk",
+            "description": "Type of seat. Usually this is Desk."
+          },
+          "host_address" : {
+            "type" : "string",
+            "example" : "584",
+            "description": "Address of host."
+          },
+          "building_name" : {
+            "type" : "string",
+            "example" : "IOE (20 B-Way)",
+            "description": "A short but human-readable name of the building that the sensor is in."
+          },
+          "room_description" : {
+            "type" : "string",
+            "example" : "",
+            "description": "A short but human-readable name of the room that the sensor is in."
+          },
+          "last_trigger_type" : {
+            "type" : "string",
+            "example" : "Occupied",
+            "description": "The last event the sensor recorded. Possible known values are Absent and Occupied, but this is not guaranteed. NB: only returned when return_states=true."
+          },
+          "survey_id" : {
+            "type" : "string",
+            "example" : "46",
+            "description": "The ID of the library/survey that was requested in the query parameter."
+          },
+          "room_type" : {
+            "type" : "string",
+            "example" : "Open Plan",
+            "description": "The type of the room."
+          },
+          "room_name" : {
+            "type" : "string",
+            "example" : "324",
+            "description": "The name of the room."
+          },
+          "room_id" : {
+            "type" : "string",
+            "example" : "291",
+            "description": "The ID of the room."
+          },
+          "location" : {
+            "type" : "string",
+            "example" : "",
+            "description": "The location of this sensor."
+          },
+          "survey_device_id" : {
+            "type" : "string",
+            "example" : "8420",
+            "description": "The ID of this survey device."
+          },
+          "share_id" : {
+            "type" : "string",
+            "example" : "None",
+            "description": "The Share ID."
+          },
+          "x_pos" : {
+            "type" : "string",
+            "example" : "32432.0",
```

Comment:
This might be a mistake in the API but does this return a string rather than a number? (same for `survey_device_id`)

Reply:
Yeah looks like we're returning a string in the API, again using the value straight from Occupeye